client/src/Journal.jsx

```diff
@@ -1 +1,25 @@
-export default function Journal() {}
+import NavBar from './NavBar';
+// import EntryView from "./EntryView";
+
+export default function Journal() {
+  return (
+    <>
+      <header className="header purple-background">
```

Comment:
FYI 💡: You could to make this cleaner, have a parent `<PageHeader />` component!